soap/src/java/com/zimbra/soap/account/type/AccountZimletHostConfigInfo.java

```diff
@@ -86,6 +87,7 @@ public void addProperty(AccountZimletProperty property) {
     @GraphQLQuery(name=GqlConstants.NAME, description="Name")
     @Override
     public String getName() { return name; }
+    @GraphQLQuery(name=GqlConstants.ZIMLET_HOST_CONFIG_PROPERTIES, description="properties")
```

Comment:
provide relevant description
```suggestion
    @GraphQLQuery(name=GqlConstants.ZIMLET_HOST_CONFIG_PROPERTIES, description="zimlet host config properties")
```